fortnite_python/base.py

```diff
@@ -40,6 +42,7 @@ def __init__(self, api_key):
     }
 
     def request(self, endpoint):
+        """Creates response with parameters of self and endpoint and handles exceptions"""
```

Comment:
This function does the request to the api with the endpoint provided and returns de json (if the response is ok)